sqlglot/generator.py

```diff
@@ -384,7 +396,16 @@ def tablealias_sql(self, expression):
         return f"{alias}{columns}"
 
     def bitstring_sql(self, expression):
-        return f"b'{self.sql(expression, 'this')}'"
+        if not self.bit_string_start:
+            return f"{self.sql(expression, 'this')}"
```

Comment:
Converting to decimal seems like the right thing to do for dialects that don't support a special format. Do you agree?

Reply:
why is it a decimal?

Reply:
Because the dialect doesn't support hexademcial notation

Reply:
no i understand that but why decimal? i don't understand how bit strings convert to decimals?

Reply:
Discussed in slack. Decimal only for the dialects that don't support hexadecimal or bit notation.